docusaurus/docs/actors/appgate_server.md

```diff
@@ -0,0 +1,33 @@
+---
+title: AppGate Server
+sidebar_position: 6
+---
+
+# AppGate Server <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+- [CLI](#cli)
+
+## Overview
+
+An `AppGate Server` is responsible for relaying requests between a client dApp
+(e.g. mobile app, web app, etc...) and the `Supplier`s on Pocket Network, handling
+all intermediary business logic.
+
+A `Gateway` operator or a sovereign `Application` interested in accessing Pocket
```

Comment:
What do you think about having `Gateway`, `Application` and `Supplier` mentions as links to their respective pages?